Gems/ROS2/Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -429,6 +428,10 @@ namespace ROS2
             auto prefabInterface = AZ::Interface<AzToolsFramework::Prefab::PrefabPublicInterface>::Get();
             [[maybe_unused]] auto createPrefabOutcome =
                 prefabInterface->InstantiatePrefab(relativePath.String(), AZ::EntityId(), AZ::Vector3::CreateZero());
+            if (createPrefabOutcome.IsSuccess())
+            {
+                MoveEntityToDefaultSpawnPoint(createPrefabOutcome.GetValue(), m_spawnPosition);
```

Comment:
User selects the spawn point out of a combo box when using the importer - it probably should not be named "default" in such case.

Reply:
There are two positions that can be called default. One is the 0,0,0 point. I've changed the name of it to simulation origin. The second is the default spawning position of the spawn point. In the component, it is called default and can be other than 0,0,0.